cmd/klotho/main.go

```diff
@@ -83,11 +84,7 @@ func main() {
 
 	err := root.Execute()
 	if err != nil {
-		if cfg.internalDebug {
```

Comment:
i was confused on why we have this. just reprints errors it looks like so i removed it. If theres a reason to have it ill add it back and have to change my implementation

Reply:
The difference is in the format: `%+v` vs `%v`. The `+` gives a full stack trace, which is only relevant for us developers and not for users.

Reply:
so do we want to have it as a cli flag then? because this could be exposed to the end user right?